### PR TITLE
Add Trace ID aware load-balancing exporter (loadbalancingexporter)

### PR DESCRIPTION
**Pull request by vastin**
**Description:**
Add loadbalancingexporter to help mitigate single goroutine pipeline problem mentioned at https://github.com/open-telemetry/opentelemetry-collector/issues/2248

**Testing:** 
local test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ This table represents the supported components of AWS OTel Collector in 2020. Th
 |                                 |                               | newrelicexporter                   |                        |
 |                                 |                               | sapmexporter                       |                        |
 |                                 |                               | signalfxexporter                   |                        |
+|                                 |                               | loadbalancingexporter              |                        |
```
Comment (mxiamxia) [inline README.md:41]: We need to go thru AWS AppSec for any new components will be added into ADOT.

Comment (vastin) [reply README.md:41]: OK. I will split unit test fixing and create another PR to fix unit test in go1.16.

Comment (mxiamxia) [reply README.md:41]: Thanks, do you think we can merge the new component support to `prometheus` branch until it's ready for the release?

Comment (vastin) [reply README.md:41]: Yes. I have changed this PR to `premetheus` branch.

**Pull request closed without merging**